src/app/components/account/details/programatic-usage/aad-credentials-picker/generate-aad-app-secret/generate-aad-app-secret.html

```diff
@@ -31,11 +31,11 @@ <h4>Secret</h4>
         <bl-aad-app-secret-picker [formControl]="secret">
         </bl-aad-app-secret-picker>
 
-        <bl-button type="wide" color="primary" [action]="addNewSecret" matTooltip="Add a new secret without modifing the existing ones">
+        <bl-button type="wide" color="primary" [action]="addNewSecret" title="AB#558 Add a new secret without modifing the existing ones">
```

Comment:
Looks like a copy/paste error or something with the title

Reply:
Oops, that was a little temporary note to myself.